_posts/2022-10-19-quarkus-observability-roadmap-2023.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: 'Quarkus Observability Roadmap 2023'
+date: 2022-10-19
+tags: observability tracing metrics opentelemetry smallrye microprofile
+synopsis: 'Quarkus Observability Roadmap 2023.'
+author: radcortez
+---
+
+https://quarkus.io[Quarkus] had come a long way in the Observability space. Initial support for Tracing and Metrics
+came from MicroProfile and their respective implementations,
+https://github.com/smallrye/smallrye-opentracing/[SmallRye OpenTracing] and
+https://github.com/smallrye/smallrye-metrics/[SmallRye Metrics]. These expanded beyond the original specifications, and
+support for Tracing and Metrics reached some of the most popular https://quarkus.io[Quarkus] extensions.
+
+Our goal with https://quarkus.io[Quarkus] is always to offer the best technologies to get the job done. It was true for
+both https://github.com/smallrye/smallrye-opentracing/[SmallRye OpenTracing] and
+https://github.com/smallrye/smallrye-metrics/[SmallRye Metrics], but we now believe there are better alternatives to
+pursue in the upcoming https://quarkus.io[Quarkus] versions.
+
+== OpenTracing Archived
+
+At the beginning of 2022, the https://https://www.cncf.io[CNCF] announced that it was
+https://www.cncf.io/blog/2022/01/31/cncf-archives-the-opentracing-project/[archiving the OpenTracing project], meaning
+that the project is no longer maintained. An expected decision since the merger between
+https://opentracing.io[OpenTracing] and http://opencensus.io/[OpenCensus] to form
+http://opentelemetry.io[OpenTelemetry].
+
+https://quarkus.io[Quarkus] released the first version of the OpenTelemetry Extension with Quarkus 1.13.0.Final, in Q1
+of 2021. The initial support was only for HTTP Tracing but slowly moved to other components, including CDI, gRPC,
+Reactive, Messaging, and many more, while also adopting and adjusting to all the new APIs released by OpenTelemetry
```

Comment:
Is it `Reactive, Messaging` or `Reactive Messaging`? Just checking.

Reply:
It is `Reactive, Messaging`, because it is not only Reactive Messaging, but also REST Reactive, but I thought it would be more confusing. We could name everything to make it explicit.